lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -100,6 +109,27 @@ export class LhcPeriodsOverviewModel extends Observable {
         return this._pagination;
     }
 
+    /**
+     * Getter to lhc periods names filter model
+     * @return {TextTokensFilterModel} lhc periods names filter model
+     */
+    get namesFilterModel() {
+        return this._namesFilterModel;
+    }
+
+    /**
+     * Build filter seach query phrase
+     * @return {string} filter search query phrase
+     */
+    getFilterSearchQueryPhrase() {
+        return this._namesFilterModel.normalized
+            .map((lhcPeriodName) => {
+                const uppercasedPrefix = lhcPeriodName.slice(0, 3).toUpperCase();
+                const lowercaseLettersSuffix = lhcPeriodName.slice(3).toLocaleUpperCase();
+                return `filter[names][]=${uppercasedPrefix}${lowercaseLettersSuffix}`;
+            }).join('&');
```

Comment:
I think this is overcomplicated. Why not simply sending any comma separated string to the backend (we don't use `;` but only `,` in bookkeeping, to be honest I think it is unnecessary complexitiy. Has it been asked for?) and match case-insensitively?

Reply:
for now, to use just name